README.md

```diff
@@ -69,32 +69,67 @@ The project includes mock interactions of external APIs (GPC, SDS) implemented i
 
 The folder `docker/wiremock/stubs` describes the supported interactions.
 
-## Unit test section 
+## How to run tests
+
+**Warning**: Gradle uses a [Build Cache](https://docs.gradle.org/current/userguide/build_cache.html) to re-use compile and
+test outputs for faster builds. To re-run passing tests without making any code changes you must first run 
+`./gradle clean` to clear the build cache. Otherwise, gradle uses the cached outputs from a previous test execution to 
```

Comment:
should this be gradle wrapper?

Reply:
should we add navigate to service?

Reply:
fixed typo, added a sentence about directory for gradle commands